### ISSUE TITLE
Process finished with exit code -1073741819 (0xC0000005)  在飞浆Ai Studio平台可以运行，但是windows10 不可以

### ISSUE THREAD

**Issue by immense8342** [PaddlePaddle/PaddleOCR#3356]
G:\item\1\PaddleOCR-release-2.1\venv\Scripts\python.exe G:/item2/my-paddle-ocr/tools/train.py -c G:\new\my-paddle-ocr\configs\lcls\lcls_mv3.yml
D:\software\Anaconda3\lib\site-packages\setuptools\depends.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
[2021/07/16 14:15:38] root INFO: Architecture : 
[2021/07/16 14:15:38] root INFO:     Backbone : 
[2021/07/16 14:15:38] root INFO:         model_name : small
[2021/07/16 14:15:38] root INFO:         name : MobileNetV3
[2021/07/16 14:15:38] root INFO:         scale : 0.35
[2021/07/16 14:15:38] root INFO:     Head : 
[2021/07/16 14:15:38] root INFO:         class_dim : 3
[2021/07/16 14:15:38] root INFO:         name : LClsHead
[2021/07/16 14:15:38] root INFO:     Neck : None
[2021/07/16 14:15:38] root INFO:     Transform : None
[2021/07/16 14:15:38] root INFO:     algorithm : LCLS
[2021/07/16 14:15:38] root INFO:     model_type : lcls
[2021/07/16 14:15:38] root INFO: Eval : 
[2021/07/16 14:15:38] root INFO:     dataset : 
[2021/07/16 14:15:38] root INFO:         data_dir : G:\new\my-paddle-ocr\train_data\lcls\train
[2021/07/16 14:15:38] root INFO:         label_file_list : ['G:\\new\\my-paddle-ocr\\train_data\\lcls\\test.txt']
[2021/07/16 14:15:38] root INFO:         name : SimpleDataSet
[2021/07/16 14:15:38] root INFO:         transforms : 
[2021/07/16 14:15:38] root INFO:             DecodeImage : 
[2021/07/16 14:15:38] root INFO:                 channel_first : False
[2021/07/16 14:15:38] root INFO:                 img_mode : BGR
[2021/07/16 14:15:38] root INFO:             ClsLabelEncode : None
[2021/07/16 14:15:38] root INFO:             ClsResizeImg : 
[2021/07/16 14:15:38] root INFO:                 image_shape : [3, 48, 192]
[2021/07/16 14:15:38] root INFO:             KeepKeys : 
[2021/07/16 14:15:38] root INFO:                 keep_keys : ['image', 'label']
[2021/07/16 14:15:38] root INFO:     loader : 
[2021/07/16 14:15:38] root INFO:         batch_size_per_card : 16
[2021/07/16 14:15:38] root INFO:         drop_last : False
[2021/07/16 14:15:38] root INFO:         num_workers : 0
[2021/07/16 14:15:38] root INFO:         shuffle : False
[2021/07/16 14:15:38] root INFO: Global : 
[2021/07/16 14:15:38] root INFO:     cal_metric_during_train : True
[2021/07/16 14:15:38] root INFO:     checkpoints : None
[2021/07/16 14:15:38] root INFO:     debug : False
[2021/07/16 14:15:38] root INFO:     distributed : False
[2021/07/16 14:15:38] root INFO:     epoch_num : 100
[2021/07/16 14:15:38] root INFO:     eval_batch_step : [0, 1000]
[2021/07/16 14:15:38] root INFO:     infer_img : doc/imgs_words_en/word_10.png
[2021/07/16 14:15:38] root INFO:     label_list : ['Latin', 'Chinese', 'Japanese']
[2021/07/16 14:15:38] root INFO:     log_smooth_window : 20
[2021/07/16 14:15:38] root INFO:     pretrained_model : None
[2021/07/16 14:15:38] root INFO:     print_batch_step : 10
[2021/07/16 14:15:38] root INFO:     save_epoch_step : 3
[2021/07/16 14:15:38] root INFO:     save_inference_dir : None
[2021/07/16 14:15:38] root INFO:     save_model_dir : ./output/Lcls/mv3/
[2021/07/16 14:15:38] root INFO:     use_gpu : True
[2021/07/16 14:15:38] root INFO:     use_visualdl : False
[2021/07/16 14:15:38] root INFO: Loss : 
[2021/07/16 14:15:38] root INFO:     name : LClsLoss
[2021/07/16 14:15:38] root INFO: Metric : 
[2021/07/16 14:15:38] root INFO:     main_indicator : acc
[2021/07/16 14:15:38] root INFO:     name : LClsMetric
[2021/07/16 14:15:38] root INFO: Optimizer : 
[2021/07/16 14:15:38] root INFO:     beta1 : 0.9
[2021/07/16 14:15:38] root INFO:     beta2 : 0.999
[2021/07/16 14:15:38] root INFO:     lr : 
[2021/07/16 14:15:38] root INFO:         learning_rate : 0.001
[2021/07/16 14:15:38] root INFO:         name : Cosine
[2021/07/16 14:15:38] root INFO:     name : Adam
[2021/07/16 14:15:38] root INFO:     regularizer : 
[2021/07/16 14:15:38] root INFO:         factor : 0
[2021/07/16 14:15:38] root INFO:         name : L2
[2021/07/16 14:15:38] root INFO: PostProcess : 
[2021/07/16 14:15:38] root INFO:     name : LClsPostProcess
[2021/07/16 14:15:38] root INFO: Train : 
[2021/07/16 14:15:38] root INFO:     dataset : 
[2021/07/16 14:15:38] root INFO:         data_dir : G:\new\my-paddle-ocr\train_data\lcls\train
[2021/07/16 14:15:38] root INFO:         label_file_list : ['G:\\new\\my-paddle-ocr\\train_data\\lcls\\train.txt']
[2021/07/16 14:15:38] root INFO:         name : SimpleDataSet
[2021/07/16 14:15:38] root INFO:         transforms : 
[2021/07/16 14:15:38] root INFO:             DecodeImage : 
[2021/07/16 14:15:38] root INFO:                 channel_first : False
[2021/07/16 14:15:38] root INFO:                 img_mode : BGR
[2021/07/16 14:15:38] root INFO:             ClsLabelEncode : None
[2021/07/16 14:15:38] root INFO:             RecAug : 
[2021/07/16 14:15:38] root INFO:                 use_tia : False
[2021/07/16 14:15:38] root INFO:             RandAugment : None
[2021/07/16 14:15:38] root INFO:             ClsResizeImg : 
[2021/07/16 14:15:38] root INFO:                 image_shape : [3, 48, 192]
[2021/07/16 14:15:38] root INFO:             KeepKeys : 
[2021/07/16 14:15:38] root INFO:                 keep_keys : ['image', 'label']
[2021/07/16 14:15:38] root INFO:     loader : 
[2021/07/16 14:15:38] root INFO:         batch_size_per_card : 16
[2021/07/16 14:15:38] root INFO:         drop_last : True
[2021/07/16 14:15:38] root INFO:         num_workers : 0
[2021/07/16 14:15:38] root INFO:         shuffle : False
[2021/07/16 14:15:38] root INFO: train with paddle 2.1.1 and device CUDAPlace(0)
[2021/07/16 14:15:38] root ERROR: ['G:\\new\\my-paddle-ocr\\train_data\\lcls\\train.txt'],list len长度：1
[2021/07/16 14:15:38] root INFO: Initialize indexs of datasets:['G:\\new\\my-paddle-ocr\\train_data\\lcls\\train.txt']
[2021/07/16 14:15:38] root ERROR: label_file_list:, ratio_list:执行到这里啦！
[2021/07/16 14:15:38] root ERROR: data_idx_order_list执行到这里啦！
[2021/07/16 14:15:38] root ERROR: do_shuffle执行到这里啦！
[2021/07/16 14:15:38] root ERROR: create_operators执行到这里啦！
[2021/07/16 14:15:38] root INFO: train_dataloader 成功
[2021/07/16 14:15:38] root ERROR: ['G:\\new\\my-paddle-ocr\\train_data\\lcls\\test.txt'],list len长度：1
[2021/07/16 14:15:38] root INFO: Initialize indexs of datasets:['G:\\new\\my-paddle-ocr\\train_data\\lcls\\test.txt']
[2021/07/16 14:15:38] root ERROR: label_file_list:, ratio_list:执行到这里啦！
[2021/07/16 14:15:38] root ERROR: data_idx_order_list执行到这里啦！
[2021/07/16 14:15:38] root ERROR: do_shuffle执行到这里啦！
[2021/07/16 14:15:38] root ERROR: create_operators执行到这里啦！
[2021/07/16 14:15:38] root INFO: valid_dataloader 成功
W0716 14:15:38.567683 13524 device_context.cc:404] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.2, Runtime API Version: 11.2
W0716 14:15:38.576681 13524 device_context.cc:422] device: 0, cuDNN Version: 8.1.
[2021/07/16 14:15:41] root INFO: build model 正在
[2021/07/16 14:15:41] root INFO: train from scratch
[2021/07/16 14:15:41] root INFO: train dataloader has 1895 iters
[2021/07/16 14:15:41] root INFO: valid dataloader has 843 iters
[2021/07/16 14:15:41] root INFO: During the training process, after the 0th iteration, an evaluation is run every 1000 iterations
[2021/07/16 14:15:41] root ERROR: ['G:\\new\\my-paddle-ocr\\train_data\\lcls\\train.txt'],list len长度：1
[2021/07/16 14:15:41] root INFO: Initialize indexs of datasets:['G:\\new\\my-paddle-ocr\\train_data\\lcls\\train.txt']
[2021/07/16 14:15:41] root ERROR: label_file_list:, ratio_list:执行到这里啦！
[2021/07/16 14:15:41] root ERROR: data_idx_order_list执行到这里啦！
[2021/07/16 14:15:41] root ERROR: do_shuffle执行到这里啦！
[2021/07/16 14:15:41] root ERROR: create_operators执行到这里啦！
[2021/07/16 14:15:41] root ERROR: 开始读取数据Ing

[2021/07/16 14:15:41] root ERROR: 在这里读取数据Ing111

Process finished with exit code -1073741819 (0xC0000005)

这个是Ai studio链接
[https://aistudio.baidu.com/aistudio/projectdetail/2116085](
https://aistudio.baidu.com/aistudio/projectdetail/2116085)

一样的文件。。。。放下来本地电脑就运行不了。 =. = 。无法训练
是操作系统的原因吗？

**Comment by tink2123** [comment]
可能和环境有关，确认一下本地机器的显存是否够用，另外有条件建议在docker中运行代码。
